### PR TITLE
ui tabs component

### PR DESCRIPTION
**Pull request by KatarzynaStefanowiczPagepro**
## Description

- [] TODO write description

**New stuff** ✨

* New stuff goes here

**Changes** 🏗

* Changes go here

**Deletions** ⚰️

* Deletions go here

## TODO

- [ ] Add TODOs

(Resolves | Contributes to) #31415

### CODE REVIEW

File: src/components/shared/Extensions/Tabs/index.tsx [hunk 1]
```diff
@@ -0,0 +1 @@
+export { default } from './Tabs';
```
Comment (krawczykkatarzyna) [inline src/components/shared/Extensions/Tabs/index.tsx:1]: change extension to `.ts`

File: src/components/shared/Extensions/Tabs/Tabs.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+import { Tabs as ReactTabs, Tab, TabScreen } from 'react-tabs-scrollable';
+import { useIntl } from 'react-intl';
+import { TabsProps } from './types';
+import styles from './Tabs.module.css';
+
+const displayName = 'Extensions.Tabs';
+
+const Tabs: FC<TabsProps> = ({ items }) => {
+  const { formatMessage } = useIntl();
+  const [activeTab, setActiveTab] = React.useState(1);
```
Comment (krawczykkatarzyna) [inline src/components/shared/Extensions/Tabs/Tabs.tsx:11]: active tab should't be hardcoded

File: src/components/common/Extensions/ImageCarousel/consts.tsx [hunk 3]
```diff
@@ -1,5 +1,5 @@
 export const images = [
-  'https://res.cloudinary.com/ifeomaimoh/image/upload/v1652345767/demo_image2.jpg',
-  'https://res.cloudinary.com/ifeomaimoh/image/upload/v1652345874/demo_image1.jpg',
-  'https://img.freepik.com/free-vector/hand-painted-watercolor-pastel-sky-background_23-2148902771.jpg?w=2000',
+  'https://source.unsplash.com/450x300/?oahu?r=1',
```
Comment (krawczykkatarzyna) [inline src/components/common/Extensions/ImageCarousel/consts.tsx:2]: why that change? PR related to Tabs component not carousel

File: src/components/shared/Extensions/Tabs/index.tsx (hunk 1 above)
Comment (KatarzynaStefanowiczPagepro) [reply src/components/shared/Extensions/Tabs/index.tsx]: Fixed

File: src/components/shared/Extensions/Tabs/Tabs.tsx (hunk 2 above)
Comment (KatarzynaStefanowiczPagepro) [reply src/components/shared/Extensions/Tabs/Tabs.tsx]: Fixed

File: src/components/common/Extensions/ImageCarousel/consts.tsx (hunk 3 above)
Comment (KatarzynaStefanowiczPagepro) [reply src/components/common/Extensions/ImageCarousel/consts.tsx]: Changes are reverted

**Review by krawczykkatarzyna — APPROVED** [review]

**Pull request merged**